README.md

```diff
@@ -189,6 +189,30 @@ Check out the [Firebase example](https://homebaseio.github.io/homebase-react/#!/
 
 ##  Debugging tips
 
+### Custom chrome formatters
```

Comment:
@smothers this looks great. Can we add something like this to the documentation website as well?

Reply:
just did. I thought it was pretty important so I stuck it on the front page (Overview)